packages/client/src/workflow-client.ts

```diff
@@ -164,6 +164,11 @@ export interface WorkflowHandle<T extends Workflow = Workflow> extends BaseWorkf
     options?: WorkflowUpdateOptions & { args?: Args }
   ): Promise<WorkflowUpdateHandle<Ret>>;
 
+  /**
+   * Get a handle to an Update.
+   */
+  getUpdateHandle<Ret>(updateId: string, workflowId: string, workflowRunId?: string): WorkflowUpdateHandle<Ret>;
```

Comment:
`firstExecutionRunId`? Should probably add in an optional options object, like we did on `getHandle()`.

Reply:
Good call, I've added the options object. Shall we leave `firstExecutionRunId` for a subsequent PR? I don't have any test coverage for it yet.

Reply:
Yeah, leave it out for now. I'm not totally sure that this is pertinent, as the update handle represents an update that was sent to a specific workflow run... 🤔 I'd have to think a bit more about this, but don't have time for this ATM.